packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -712,7 +728,29 @@ exports.extendNodeType = ({ type, store, reporter }) => {
     }
 
     try {
-      const absolutePath = await cacheImage(store, image, options, reporter)
+      const {
+        file: { contentType, url: imgUrl, fileName },
+      } = image
+
+      if (contentType.indexOf(`image/`) !== 0) {
+        return null
+      }
+
+      // 256px should be enough to properly detect the dominant color
+      if (!options.width) {
+        options.width = 256
+      }
+
+      const extension = mimeTypeExtensions.get(contentType)
+      const url = `https:` + createUrl(imgUrl, options)
```

Comment:
Why don't we add https in the createUrl?

Reply:
@wardpeet because this function is used on browser side as well. That way it works on http as well.

Reply:
This goes to contentful cdn right? Than we should force https. There is no reason to do a HTTP call ever if the server supports https.

Reply:
Ahh very true. Using http images on https sides causes issues. But this would be the other way around. 

Will change it!

Reply:
Done